mne/decoding/transformer.py

```diff
@@ -81,7 +81,7 @@ class Scaler(TransformerMixin, BaseEstimator):
     info : instance of Info | None
         The measurement info. Only necessary if ``scalings`` is a dict or
         None.
-    scalings : dict, string, default None.
+    scalings : dict, string, default None
```

Comment:
shouldn't this one be 
```
scalings : dict | str | None
```

Reply:
yes but let's work on that in a separate PR, there are probably many such instances in our code base. I'll make a comment above about this